File: problemas/1100/1176.py
t = int(input())
fib = [0, 1]

for i in range(t):
    n = int(input())

    while len(fib) < n + 1:
        fib.append(fib[-1] + fib[-2])

    print(f'Fib({n}) = {fib[n]}')

File: problemas/1100/1176.dart
import 'dart:io';

void main() {
  final t = int.parse(stdin.readLineSync());
  final fib = new List<BigInt>.from([BigInt.zero, BigInt.one]);

  for (var i = 0; i < t; i++) {
    final n = int.parse(stdin.readLineSync());

    while (fib.length < n + 1) {
      fib.add(fib[fib.length - 1] + fib[fib.length - 2]);
    }

    stdout.writeln('Fib($n) = ${fib[n].toUnsigned(64)}');
  }
}